sys/nginx/lua/hook.lua

```diff
@@ -1,42 +1,4 @@
-local regex = [[]]
-regex = regex .. [[/(]]
-regex = regex .. [[circleci]]
-regex = regex .. [[|circleci/.+]]
-regex = regex .. [[|circleci/.+\.svg]]
-regex = regex .. [[|circleci\.svg]]
-regex = regex .. [[|composerlock]]
-regex = regex .. [[|composerlock\.svg]]
-regex = regex .. [[|d/daily]]
-regex = regex .. [[|d/daily\.png]]
-regex = regex .. [[|d/daily\.svg]]
-regex = regex .. [[|d/monthly]]
-regex = regex .. [[|d/monthly\.png]]
-regex = regex .. [[|d/monthly\.svg]]
-regex = regex .. [[|d/total]]
-regex = regex .. [[|d/total\.png]]
-regex = regex .. [[|d/total\.svg]]
-regex = regex .. [[|dependents]]
-regex = regex .. [[|dependents\.svg]]
-regex = regex .. [[|downloads]]
-regex = regex .. [[|downloads\.png]]
-regex = regex .. [[|downloads\.svg]]
-regex = regex .. [[|gitattributes]]
-regex = regex .. [[|gitattributes\.svg]]
-regex = regex .. [[|license]]
-regex = regex .. [[|license\.png]]
-regex = regex .. [[|license\.svg]]
-regex = regex .. [[|suggesters]]
-regex = regex .. [[|suggesters\.svg]]
-regex = regex .. [[|v/stable]]
-regex = regex .. [[|v/stable\.png]]
-regex = regex .. [[|v/stable\.svg]]
-regex = regex .. [[|v/unstable]]
-regex = regex .. [[|v/unstable\.png]]
-regex = regex .. [[|v/unstable\.svg]]
-regex = regex .. [[|version]]
-regex = regex .. [[|version\.png]]
-regex = regex .. [[|version\.svg]]
-regex = regex .. [[)$]]
+local regex = [[/(circleci|circleci/.+|circleci/.+\.svg|circleci\.svg|composerlock|composerlock\.svg|d/daily|d/daily\.png|d/daily\.svg|d/monthly|d/monthly\.png|d/monthly\.svg|d/total|d/total\.png|d/total\.svg|dependents|dependents\.svg|downloads|downloads\.png|downloads\.svg|gitattributes|gitattributes\.svg|license|license\.png|license\.svg|suggesters|suggesters\.svg|v/stable|v/stable\.png|v/stable\.svg|v/unstable|v/unstable\.png|v/unstable\.svg|version|version\.png|version\.svg)$]]
```

Comment:
+100

Reply:
```
circleci | circleci/.+ | circleci/.+\.svg | circleci\.svg
```
the second one already matches everything else ...  should be simplified :)

Reply:
[[ ]] is a multiline string .. I'd take advantage of this fact to have a (simplified) block on each line .. no need for a repeated assignment but no need to a loooong unreadable line too ;)